Plugin/src/SofaPython3/Prefab.cpp

```diff
@@ -69,7 +69,7 @@ void Prefab::reinit()
     doReInit();
 
     /// Beurk beurk beurk
-    sofa::simulation::node::init(this);
+    sofa::simulation::getSimulation()->init(this);
```

Comment:
This is a revert of https://github.com/sofa-framework/SofaPython3/pull/357

I think this branch must be rebased on the current master